hack/deploy-karmada.sh

```diff
@@ -22,7 +22,7 @@ source "${REPO_ROOT}"/hack/util.sh
 
 function usage() {
   echo "This script will deploy karmada control plane to a given cluster."
-  echo "Usage: hack/deploy-karmada.sh <KUBECONFIG> <CONTEXT_NAME> [KARMADA_API_SERVER_IP]"
+  echo "Usage: hack/deploy-karmada.sh <KUBECONFIG> <CONTEXT_NAME> [REMOTE_MODE]"
   echo "Example: hack/deploy-karmada.sh ~/.kube/config karmada-host"
```

Comment:
Should we add an example with `REMOTE_MODE`?

Reply:
Ok, no problem